fluss-server/src/main/java/com/alibaba/fluss/server/kv/prewrite/KvPreWriteBuffer.java

```diff
@@ -426,4 +462,10 @@ public String toString() {
             return value == null ? "null" : "[" + Base64.getEncoder().encodeToString(value) + "]";
         }
     }
+
+    /** The reason why we truncate the kv pre-write buffer. */
+    public enum KvPreWriteBufferTruncateReason {
```

Comment:
too long, simplify it to `TruncateReason`.

Reply:
Done!